base/applications/kbswitch/kbswitch.c

```diff
@@ -8,6 +8,11 @@
  */
 
 #include "kbswitch.h"
+#include <unknwn.h>
+#include <docobj.h>
+#include <shlobj.h>
+#include <shobjidl.h>
```

Comment:
Are *all* those headers necessary? Can we remove some of these? (For example, are the first two "unknwn.h" and "docobj.h" necessary?)

Reply:
Deleted some `#include`'s.